.github/workflows/build-and-test-BASE.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        framework: [net452, net462, net472, net480, netcoreapp3.1, net5.0, net6.0]
+        framework: [net462, net472, net480, netcoreapp3.1, net5.0, net6.0]
```

Comment:
This change will make force auto-instrumentation to major upgrade for .NET. Are we going to benefit from this change?

Reply:
Discussed offline. Auto-instrumentation min version is net452. Will be following up later in the week to discuss options.